README.md

```diff
@@ -78,16 +85,31 @@ The `LiveFixtures::Import` class allows you to specify the location of your fixt
       end
     end
 
+Options may be passed when initializing an importer as follow:
+ - show_progress: defaults to true.
+   Pass false to disable the progress bar output.
+ - skip_missing_tables: defaults to false.
+   Pass true to avoid raising an error when a table listed in insert_order has
+   no yml file.
+ - skip_missing_refs: defaults to true.
```

Comment:
I would prefer if the default were `false`. I think the user can deliberately choose to be less safe, but should be able to trust a library to prevent them from shooting themselves in the foot.